src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasureRecord.test.ts

```diff
@@ -0,0 +1,99 @@
+import { migrateCalculatedMeasureRecord } from "./migrateCalculatedMeasureRecord";
+import _cloneDeep from "lodash/cloneDeep";
+
+const distinctCountCity = {
+  entry: {
+    content:
+      '{"expression":"Count(Descendants([Geography].[City].CurrentMember, [Geography].[City].[City]), EXCLUDEEMPTY)","properties":["FORMAT_STRING = \\"#,###.##\\"","CAPTION = \\"Distinct count city\\" "]}',
```

Comment:
Did you copy this from the content server ? I am surprised that there is a trailing whitespace after 'Distinct count city'.

Reply:
No, I didn't copy it from the content server, none of the calculated measures saved on the content server had any additional properties so I just made it up. It's my mistake, I made a typo, I will fix it !